### PR TITLE
goodhosts 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by mdcurran**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/goodhosts.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  desc "Simple hosts file management in a Go cli"
+  homepage "https://github.com/goodhosts/cli"
+  url "https://github.com/goodhosts/cli/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "6c956ffb6b9c838cd8e6dfe72b9e60fd8b21c674e57a0ad2fced7e23b042ce60"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"goodhosts", "."
```
Comment (SMillerDev) [inline Formula/goodhosts.rb:11]:
```suggestion
    system "go", "build", *std_go_args
```

File: Formula/goodhosts.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  test do
+    (testpath/"hosts").write <<~EOS
+      127.0.0.1 localhost
+    EOS
+
+    system "goodhosts", "-f", testpath/"hosts", "a", "127.0.0.1", "localhost", "test.localhost"
+    assert_equal "127.0.0.1 localhost test.localhost\n", shell_output("cat #{testpath}/hosts")
```
Comment (carlocab) [inline Formula/goodhosts.rb:20]:
```suggestion
    system bin/"goodhosts", "-f", testpath/"hosts", "a", "127.0.0.1", "localhost", "test.localhost"
    assert_equal "127.0.0.1 localhost test.localhost", (testpath/"hosts").read.strip
```

File: Formula/goodhosts.rb (hunk 1 above)
Comment (carlocab) [inline Formula/goodhosts.rb:11]:
```suggestion
    system "go", "build", *std_go_args
```

**Comment by SMillerDev** [comment]
Hi, this seems like a nice software, but not notable enough for us to package in Homebrew for now. This is not a decision on the merits of the software, but we cannot distribute every single piece of software and have to draw a line on “notability” somewhere. Of course, we would be happy to reconsider in the future.

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Pull request closed without merging**